challenge.md

```diff
@@ -5,19 +5,34 @@ permalink: /challenge.html
 
-This year's *Question Answering over Linked Data* challenge, better known as *QALD*, focuses on **Question Answering over Wikidata**. This will force QA systems to adapt to a new graph as QALD has been DBpedia-centric so far. We will have a novel sub-task on low-resource language QALD to drive accessibility in these languages forward. In particular, QA systems over multilingual KGs widen the access to the Web of Data to lay users. Thus, QA systems increase the accessibility of the Web in general.
+This year's Question Answering over Linked Data challenge has **2 tasks** and focuses on **Question Answering over Wikidata**. This will force QA systems to adapt to a new graph as QALD has been DBpedia-centric so far. 
 
-We also introduce a **new task on comparative QA**, where the goal is to efficiently answer factual multilingual comparative questions using knowledge basesComparative QA has a wide range of use cases, such as academic search engines or e-commerce.
+For QALD-10 task 1, we will have a novel sub-task on **low-resource language QALD (Russian, Chinese, German, English) over Wikidata** to drive accessibility in these languages forward. In particular, QA systems over multilingual KGs widen the access to the Web of Data to lay users. Thus, QA systems increase the accessibility of the Web in general.
 
-### Task 1: Multilingual Question Answering over Knowledge Graphs
+For QALD-10 task 2, we introduce a **new task on comparative QA**, where the goal is to efficiently answer factual multilingual comparative questions using knowledge bases. Comparative QA has a wide range of use cases, such as academic search engines or e-commerce.
+
+## Task 1: Multilingual Question Answering over Knowledge Graphs
 
 Given the diversity of languages used on the Web, there is an increasing need to facilitate multilingual access to semantic data. The core task of QALD is thus to retrieve answers from an RDF data repository given an information need expressed in a variety of natural languages.
 
-The underlying RDF dataset is a fixed Wikidata dump with meta-data. The evaluation, i.e. train and test, dataset are based on roughly 450 QALD-9 questions but extended by more than 100 % to satisfy also data-hungry algorithms. The questions vary with respect to their complexity and each is annotated with a manually specified SPARQL query and answers. The evaluation will be based on the established QALD-F1-measure.
+The underlying RDF dataset is a fixed Wikidata dump with meta-data (TBA). The evaluation, i.e., train and test, dataset are based on 558 [QALD-9-plus](https://github.com/Perevalov/qald_9_plus) questions. This years test set will compass more 500 novel, manually created questions to satisfy also data-hungry algorithms. The questions vary with respect to their complexity and each is annotated with a manually specified SPARQL query and answers. The evaluation will be based on the established QALD-F1-measure.
```

Comment:
> The evaluation, i.e., train and test, dataset are based on 558 [QALD-9-plus (https://github.com/Perevalov/qald_9_plus) questions.

Propose change to: "participants will have to develop their systems using 558 [QALD-9-plus (https://github.com/Perevalov/qald_9_plus) questions for training and local testing.

> This years test set will compass more 500 novel, manually created questions to satisfy also data-hungry algorithms.

Make it more precise: This years **private** test set (the one that will be used for the building the leaderboard)....